packages/cxl-lumo-styles/scss/cxl-vaadin-button.scss

```diff
@@ -1,3 +1,5 @@
+// @see https://github.com/vaadin/vaadin-button/blob/master/theme/lumo/vaadin-button-styles.html
+// @see https://github.com/vaadin/vaadin-button/blob/master/src/vaadin-button.html
```

Comment:
Structure upgrade proposal:

* Move `scss/cxl-vaadin-button.scss` -> `scss/integrations/vaadin-button-scss`
* `global.scss`: add `/** Integrations. */` block, first guest `@use "integrations/vaadin-button"`
* Move `src/cxl-vaadin-button.js` into button-only story, then drop `src/cxl-vaadin-button.js`
* You can also refactor `.headroom` to `scss/integrations/headroom.scss` to demonstrate general strategy in one commit